config/envs/unit_test.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+
+from fsd_utils import configclass
+
+from config.envs.default import DefaultConfig
+
+
+@configclass
+class UnitTestConfig(DefaultConfig):
+    FLASK_ROOT = Path(__file__).parent.parent.parent
```

Comment:
You don't need to specify this in the UnitTestConfig again as it already inherits it from DefaultConfig.